agr_literature_service/lit_processing/dump_json_data.py

```diff
@@ -0,0 +1,576 @@
+import argparse
+import logging
+from os import environ, makedirs, path, rename, remove
+from dotenv import load_dotenv
+from datetime import datetime, date
+import json
+
+from agr_literature_service.api.models import CrossReferenceModel, \
+    ReferenceModel, ModModel, ModCorpusAssociationModel, \
+    ReferenceCommentAndCorrectionModel, AuthorModel, MeshDetailModel, \
+    ResourceModel, ModReferenceTypeModel
+from agr_literature_service.lit_processing.helper_sqlalchemy import create_postgres_session
+from agr_literature_service.lit_processing.helper_s3 import upload_file_to_s3
+
+logging.basicConfig(format='%(message)s')
+log = logging.getLogger()
+log.setLevel(logging.INFO)
+
+load_dotenv()
+
+s3_bucket = 'agr-literature'
+sub_bucket = 'develop/reference/dumps/'
```

Comment:
Oh, nevermind, you're replacing it later around line 132.  That seems a bit odd to me, but it makes sense.

Reply:
I take it back, on the ENV_STATE == test case, the s3_filename still uses the env_state value from ENV_STATE, instead of skipping doing an s3 upload.  Presumably while in testing, we shouldn't upload to s3.  But if you think we should, we probably want a 'test' subbucket to avoid overwriting the develop/ subbucket with test data.

Reply:
You're right, it doesn't upload, sorry about that !